crashreporter/src/main/java/com/tealium/crashreporter/TealiumConfigCrashReporter.kt

```diff
@@ -0,0 +1,34 @@
+package com.tealium.crashreporter
+
+import com.tealium.core.ModuleFactory
+import com.tealium.core.Modules
+import com.tealium.core.Tealium
+import com.tealium.core.TealiumConfig
+
+/**
+ * Returns the Crash Reporter factory.
+ */
+val Modules.CrashReporter: ModuleFactory
+    get() = com.tealium.crashreporter.CrashReporter
```

Comment:
Do we typically add this bit in the Config extension? Isn't it usually added to the entry point of the module factory - `CrashReporter.kt`?